src/doc/gui_manual/Operators/OperatorTypes/CreateBonds_operator.rst

```diff
@@ -0,0 +1,76 @@
+.. _createbonds_operator_head:
+
+Create Bonds operator
+~~~~~~~~~~~~~~~~~~~~~
+
+The CreateBonds operator is used to specify ranges of distances for various 
+types of atoms, and use those ranges to create bonds. The default behavior of 
+this operator is to create a bonds between a Hydrogen and any other species 
+if the atoms are separated by a distance between 0.4 and 1.2 units 
+(e.g. Angstroms) and between a pair of atoms (not including H) if they are 
+between 0.4 and 1.9 units apart. This works well for organic molecules. 
+However, in :numref:`Figure %s <createbonds_example>`, the default distances 
+were not useful. In this case, the values were changed to create a bond 
+including H for distances between 0.4 and 1.5 units, and for other species 
+between 0.4 and 2.5 units. :numref:`Figure %s <createbonds_bonding>`.
```

Comment:
Grammar: The commas should be removed from these two sentence segments:
1. "...types of atoms, and use those ranges..."
2. "...distances between 0.4 and 1.5 units, and for other species 
between..."